specification/components/schemas/responses/cancelReferral/422Response.yaml

```diff
@@ -0,0 +1,22 @@
+description: |
+  Where status code 422 (Unprocessable Entity) is returned then an eRS-OperationOutcome-1 will be included in the body, as detailed below. 
+  Check diagnostics property for specific information regarding the error.
+
+  | Error code                         | Description                                                                                                                  |
+  | ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+  | MISSING_VALUE                      | A field defined as mandatory for an endpoint has not been provided. |
+  | INVALID_REQUEST_STATE              | The Request, while otherwise valid, is not in the correct state for the specific action being performed.                     |
+  | PATIENT_ERROR                      | An error occurred while retrieving the requested patient. Do not attempt again.                                              |
+  | NO_RELATIONSHIP                    | The user does not have a legitimate relationship with the request.                                                           |
+  | UNABLE_TO_CANCEL_APPOINTMENT       | System is unable to cancel the appointment. |
```

Comment:
As per Story ERSSUP-58662,  It is expected to return 422 if PCS rejects the booking cancellation. 

10.2. If the Appointment Booking cannot be Cancelled in the Provider Clinical System because the Provider Clinical System rejects the cancellation, then an Outcome Key of UNABLE_TO_CANCEL_APPOINTMENT and HTTP Response Code of 422 should be returned.

It is same case for Cancel Booking as well i.e. ERSSUP-57332